projects/dsp-ui/src/lib/action/pipes/formatting/knoradate.pipe.ts

```diff
@@ -6,19 +6,36 @@ import { KnoraDate } from '@dasch-swiss/dsp-js';
 })
 export class KnoraDatePipe implements PipeTransform {
 
-    transform(date: KnoraDate, format?: string): string {
+    formattedString: string;
+
+    transform(date: KnoraDate, format?: string, precisionOptions?: 'era' | 'calendar' | 'all'): string {
```

Comment:
`precisionOptions` should be called differently because this is about whether to show the era and/or calendar of a given date or period, not about precision.

Reply:
fixed in b40ef39